packages/nimble-components/src/table-column/number-text/models/tests/number-text-unit-format.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { parameterizeSpec } from '@ni/jasmine-parameterized';
+import { NumberTextFormat } from '../../types';
+import { NumberTextUnitFormat } from '../number-text-unit-format';
+import { byteUnitScale } from '../../../../utilities/unit-format/unit-scale/byte-unit-scale';
+import type { UnitScale } from '../../../../utilities/unit-format/unit-scale/unit-scale';
+
+describe('NumberTextUnitFormat', () => {
+    const testCases: readonly {
+        name: string,
+        locale?: string,
+        format: NumberTextFormat,
+        decimalDigits?: number,
+        decimalMaximumDigits?: number,
+        unitScale?: UnitScale,
+        number: number,
+        expected: string
+    }[] = [
```

Comment:
Please make these test cases more explicit like the following commit: https://github.com/ni/nimble/pull/1764/commits/ec62572289b7312557dbe31f33580bfff339daf3

- just define each property in each case and then you can avoid this typing. It improves test understanding, I don't have to jump back and forth from cases to test code to see what parameters are missing and what the default values are 
- notice in that commit how with the properties defined in each case you don't run into the type behavior of an absent property vs a property of value undefined.